examples/connectivity/plot_extract_regions_dictlearn_maps.py

```diff
@@ -23,28 +27,28 @@
 confounds = adhd_dataset.confounds
 
 ################################################################################
-# Canonical ICA decomposition of functional datasets by importing CanICA from
-# decomposition module
-from nilearn.decomposition import CanICA
-
-# Initialize canica parameters
-canica = CanICA(n_components=5, smoothing_fwhm=6.,
-                memory="nilearn_cache", memory_level=2,
-                random_state=0)
+# Import dictionary learning algorithm from decomposition module and call the
+# object and fit the model to the functional datasets
+from nilearn.decomposition import DictLearning
+
+# Initialize DictLearning object and parameters
```

Comment:
I would be inclined to think that initializing a class instance does imply that its parameters/variable are also initialized. Thus "and parameters" might perhaps be confusing to some.